indsl/validations.py

```diff
@@ -35,5 +35,5 @@ def validate_timedelta(timedelta: pd.Timedelta) -> None:
     if timedelta is pd.NaT:
         raise UserValueError("Timedelta is invalid (NaT).")
 
-    if timedelta.value < 0:
-        raise UserValueError("Timedelta must be strictly positive. The smallest possible value is '1ns'")
+    if timedelta.value <= 0:
+        raise UserValueError("Timedelta must be strictly positive. The smallest possible value is '1s'")
```

Comment:
I think the smallest value is still 1ns as inferred from the doc
https://pandas.pydata.org/docs/reference/api/pandas.Timedelta.html

Reply:
@tuanng-cognite I agree that the timedelta can be defined as 1ns from pandas' side, but this is the error message we give to the user when using the function resample_to_granularity, which can only handle units down to 1s (if I got it correctly). That is why I changed it.

Reply:
`Valid time units are day or d, hour h, minute or m and second or s` this is for `granularity` input of `resample_to_granularity`. But `validate_timedelta` is not specific for `resample_to_granularity`, it is used in other places as well.

If we want to enforce minimum 1s, we should do that inside `resample_to_granularity`. But I don't think it is this task purpose.

Reply:
Aha, ofc - did not realize the validate_timedelta function is being used by other functions that might allow 1ns. Will update 👍